addon/styles/_frost-toggle-button.scss

```diff
@@ -0,0 +1,84 @@
+.frost-toggle-button {
+  display: inline-flex;
+  justify-content: center;
+  align-items: center;
+
+  &.disabled {
+    opacity: 0.4;
+  }
+
+  .frost-toggle {
+    display: none;
+
+    &:checked {
+
+      +.frost-toggle-btn:after {
+        left: 50%;
+      }
+
+      +.frost-toggle.frost-toggle-btn {
+        background-color: $frost-color-lgrey-1;
+      }
+    }
+  }
+
+  .frost-toggle-btn.frost-toggle-disabled {
+    cursor: not-allowed;
```

Comment:
I always disliked that cursor, let's remove it for now and we can discuss with UX if it's necessary